packages/clerk-js/src/ui/components/OrganizationList/OrganizationListPage.tsx

```diff
@@ -227,3 +212,86 @@ const OrganizationListPageList = (props: { onCreateOrganizationClick: () => void
     </>
   );
 };
+
+const ForceOrganizationSelectionFlow = ({ hasOrganizationResources }: { hasOrganizationResources: boolean }) => {
+  const sessionTasksContext = useContext(SessionTasksContext);
+  const { navigateAfterCreateOrganization, hideSlug } = useOrganizationListContext();
+  const { isLoading: isLoadingOrganizationResources } = useOrganizationListInView();
+
+  const [isNavigatingAfterOrgCreation, setIsNavigatingAfterOrgCreation] = useState(false);
+  const [isCreateOrganizationFlow, setIsCreateOrganizationFlow] = useState(!hasOrganizationResources);
+
+  const isLoading = isNavigatingAfterOrgCreation || isLoadingOrganizationResources;
+  if (isLoading) {
+    return (
+      <FlowCard>
+        <FlowLoadingState />
+      </FlowCard>
+    );
+  }
+
+  return (
+    <FlowCard>
+      {isCreateOrganizationFlow ? (
+        <Box
+          sx={t => ({
+            padding: `${t.space.$none} ${t.space.$5} ${t.space.$5}`,
+          })}
+        >
+          <CreateOrganizationForm
+            flow='organizationList'
+            onComplete={sessionTasksContext?.nextTask}
+            startPage={{ headerTitle: localizationKeys('organizationList.createOrganization') }}
+            skipInvitationScreen
+            navigateAfterCreateOrganization={org => {
+              // During a force organization selection flow, keep displaying the creation form in a loading state.
+              // This allows the client-side navigation to complete before transitioning away from this view.
+              setIsNavigatingAfterOrgCreation(true);
+              return navigateAfterCreateOrganization(org);
+            }}
+            onCancel={
+              isCreateOrganizationFlow && hasOrganizationResources
+                ? () => setIsCreateOrganizationFlow(false)
+                : undefined
+            }
+            hideSlug={hideSlug}
+          />
+        </Box>
+      ) : (
+        <OrganizationListPageList onCreateOrganizationClick={() => setIsCreateOrganizationFlow(true)} />
+      )}
+    </FlowCard>
+  );
+};
+
+const FlowCard = ({ children }: PropsWithChildren) => {
+  const card = useCardState();
+
+  return (
+    <Card.Root>
+      <Card.Content sx={t => ({ padding: `${t.space.$8} ${t.space.$none} ${t.space.$none}` })}>
+        <Card.Alert sx={t => ({ margin: `${t.space.$none} ${t.space.$5}` })}>{card.error}</Card.Alert>
+        {children}
+      </Card.Content>
+      <Card.Footer />
+    </Card.Root>
+  );
+};
+
+const FlowLoadingState = () => (
+  <Flex
+    direction={'row'}
+    align={'center'}
+    justify={'center'}
+    sx={t => ({
+      height: '100%',
+      minHeight: t.sizes.$60,
+    })}
```

Comment:
Do we need this height/min-height defined? Seeing other usages where that is not the case in loading card states. https://github.com/clerk/javascript/blob/b975d96fa01ad065f98f43246c088866b4c35dec/packages/clerk-js/src/ui/components/SignIn/ResetPasswordSuccess.tsx#L24-L33